incubator/schema-registry-ui/OWNERS

```diff
@@ -0,0 +1,25 @@
+approvers:
```

Comment:
Don't put all the chart maintainers in there. You should also add yourself as approver. However, for the mechanism to work, you'd have to become a member of the Kubernetes org on Github.

See https://github.com/kubernetes/charts#owning-and-maintaining-a-chart